scss/blocks/figma-component-props-table.scss

```diff
@@ -0,0 +1,120 @@
+:root {
+    --figma-component-props-table-background-color: rgba(244, 245, 247, 0.95);
+    --figma-component-props-table-badge-border-color: #ECEEF3;
+}
+
+table.props-table {
```

Comment:
So the data table that is in the repos would not be suitable for this right?

Reply:
I created a new one instead of overriding and trying to adjust the existing one - imo it's a faster and cleaner option